packages/extension-browser/src/devtools/utils/network.ts

```diff
@@ -79,10 +79,45 @@ const generateFetchEnd = async (entry: chrome.devtools.network.Request) => {
     hops.delete(url);
 };
 
+/**
+ * Checks if a request is comming from a Service|Web worker using the
+ * `referer` header to identify it.
```

Comment:
Probably good to include *why* we want to ignore requests from workers somewhere in the comments. Otherwise looks good.

Reply:
This is explained in line 118